packages/devtools_app/lib/src/debugger/flutter/call_stack.dart

```diff
@@ -61,27 +61,49 @@ class _CallStackState extends State<CallStack> {
     final subtleStyle = TextStyle(color: theme.unselectedWidgetColor);
     final selectedStyle = TextStyle(color: theme.textSelectionColor);
 
+    Widget child;
+
+    final asyncMarker = frame.frame.kind == FrameKind.kAsyncSuspensionMarker;
+    if (asyncMarker) {
+      child = Row(
+        children: [
+          const SizedBox(width: defaultSpacing, child: Divider()),
+          Padding(
+            padding: const EdgeInsets.symmetric(horizontal: densePadding),
+            child: Text(
+              _descriptionFor(frame),
+              style: selected ? selectedStyle : subtleStyle,
+            ),
+          ),
+          const Expanded(child: Divider()),
+        ],
+      );
+    } else {
+      child = RichText(
+        maxLines: 1,
+        overflow: TextOverflow.ellipsis,
+        text: TextSpan(
+          text: _descriptionFor(frame),
+          style: selected ? selectedStyle : regularStyle,
+          children: [
+            if (!asyncMarker)
```

Comment:
this if check shouldn't be necessary since we are in the else statement of `if (asyncMarker)`

Reply:
removed